docs/getting_started.rst

```diff
@@ -0,0 +1,79 @@
+
+.. py:currentmodule:: rticonnextdds_connector
+
+Getting Started
+===============
+
+Installing RTI Connector for Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure you have Python installed. Then use *pip* to install *RTI Connector*
+as follows:
+
+.. code:: bash
+
+   $ pip install rticonnextdds_connector
+
+You can also clone the repository:
+
+.. code:: bash
+
+   $ git clone --recursive https://github.com/rticommunity/rticonnextdds-connector-py.git
+
+Running the examples
+~~~~~~~~~~~~~~~~~~~~
+
+To run the examples, first clone the repository as indicated in the previous section,
+then run any of the python scripts in the ``examples/python`` directory:
+
+.. code:: bash
+
+    python examples/python/simple/reader.py
+
+On another shell:
+
+.. code:: bash
+
+    python examples/python/simple/writer.py
+
+This is how ``reader.py`` looks like:
+
+.. literalinclude:: ../examples/python/simple/reader.py
+    :lines: 19-
+
+
+And this is ``writer.py``:
+
+.. literalinclude:: ../examples/python/simple/writer.py
+    :lines: 17-
+
+To learn more about *RTI Connector* continue to the next section,
+:ref:`Using a Connector`.
+
+Supported Platforms
+~~~~~~~~~~~~~~~~~~~
+
+*RTI Connector* has been tested with Python 2.7.14 and 3.6.3.
+
+*RTI Connector* uses a native C library. It has been tested on the following
+platforms:
+
+- Windows (64-bit Windows 10 with VS 2015 and 32-bit Windows 7 with VS 2017)
+- Linux (64-bit CentOS 6.5 with gcc 4.8.2, 32-bit Ubuntu 16.04 gcc 5.4.0, and
```

Comment:
Is there any way we can list other architectures that it works on? (e.g., the arch I use isn't listed). If not it sounds quite limited in support. Or add a sentenace explaining that if your architecture isn't here it doesn't necessarily mean that you can't use Connector

Reply:
Agreed. I can probably say Connector works on most Linux, Windows and MacOs systems. It has been tested on the following systems:

Reply:
I'm changing the language a bit, but I think PM should review this. And we need to test on Windows